docs/portal/transports/add-transport.md

```diff
@@ -20,15 +20,15 @@ The <strong> Transports </strong> page can be reached simply by clicking "Transp
-|Transfer Strategy     |<li><strong> Default </strong>： Low cost and high efficiency. </li> <li><strong> Interactive </strong>：For applications requiring better response time, HDT internal routing would be enabled in this mode. </li> <li><strong> High Concurrency </strong>：Suitable for small data and high concurrency. Normally we consider the concurrency is high when its value is higher than 1000 connections, it’s low when its value is lower than 100 connections. </li> <li><strong> High Real-time </strong>：Suitable for applications which deliver small data with low concurrency（fewer than 100 connections） and requiring  optimized response time. </li> |
+| Carry Client IP      | <li><strong> NO </strong>: Disable the function. By default, HDT will try to add the X-Forwarded-For header into the request to carry the client’s real IP address for HTTP and HTTPs application. (For HTTPs applications, the client certificate must be deployed at the HDT servers. Please contact HDT support team if you need to deploy a client certificate.) </li> <li><strong> TCP Option 0x4e </strong>: HDT establishes a TCP connection with the origin server and sends the client's real IP address to the origin server through the tcp-option 0x4e in the TCP message; the origin server can obtain the client's real IP address through the F5 device or the Linux kernel module provided by HDT.</li> <li><strong>proxy protocol v1</strong>: Using proxy protocol v1, HDT establishes a TCP connection with the origin server, and sends the client's real IP address to the origin server in the first packet; the origin server obtains the client's real IP address by parsing the proxy protocol v1 message.</li><li><strong>proxy protocol v2 </strong>: Using proxy protocol v2, HDT establishes a TCP connection with the origin server, and sends the client's real IP address to the origin server in the first packet; the origin server obtains the client's real IP address by parsing the proxy protocol v2 message.</li>|
+|Transfer Strategy     |<li><strong> Default </strong>: Low cost and high efficiency. </li> <li><strong> Interactive </strong>: For applications requiring better response time, HDT internal routing is enabled in this mode. </li> <li><strong> High Concurrency </strong>：Suitable for small data transfers and high concurrency. Normally, concurrency is considered to be high when its value exceeds 1,000 connections and low when its value is below than 100 connections. </li> <li><strong> High Real-time </strong>: Suitable for applications that deliver small data with low concurrency (fewer than 100 connections) and require optimized response time. </li> |
```

Comment:
In the second sentence, please change "response time" to "response times."
In the sentence"...when its value is below than 100 connections."" please remove the word "than."
In the last sentence, please consider replacing "and require optimized response time." with "and require optimized response times."